en/docs/remote-apps/connect-remote-applications.md

```diff
@@ -0,0 +1,118 @@
+# Connect Remote Applications
+
+If you need to observe an external application that you have written in [Ballerina programming language](https://ballerina.io/), you can connect it to Choreo.
+
+## Prerequisites
+
+Create an application in Ballerina programming language. For instructions, see [Ballerina Documentation - Writing Your First Ballerina Program](https://ballerina.io/learn/user-guide/getting-started/writing-your-first-ballerina-program/).
+
+!!! Tip
+    In this guide, we are using the same Ballerina application that is created in the guide linked above.
+
+## Step 1: Generate and copy the Ballerina configurations
+
+In this step, you will generate the Ballerina configurations that include an application secret to link the remote application to Choreo.
+
+To get these configurations, follow this procedure:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+
+2. Go to the **Remote Apps** card, and click **Connect App**.
+
+3. Click **Connect**.
+
+4. In the **Connect existing application** card, enter a name for the remote application (e.g., `HelloService`).
+
+5. Click **Next**. The Ballerina configurations with the application secret generated for the remote application are displayed.
+
+6. Copy the Ballerina configurations.
+
+Now you have the application secret that you need to set up the Ballerina project of your remote application.
+
+## Step 2: Set up the Ballerina project
+
+In this step, you will set up your Ballerina project with the configurations that connect the Ballerina application to Choreo. Here, the Ballerina project is the directory that contains the definition of your Ballerina application in a `.bal` file and all the configuration files required to run it.
+
+To set up the Ballerina Project:
+
+1. Create a directory in your machine (e.g., named `HelloWorld`) and move your file with the remote ballerina application (in this example the `hello_service.bal` file) to it.
+
+    !!! info
+        This directory will be referred to as `<BALLERINA_PROJECT>`from here onward.
+
+2. To link the Ballerina application to Choreo via the application secret, open a new file in a text editor and add the Ballerina configurations you copied in Step 1. Then save it as `Config.toml` in the `<BALLERINA_PROJECT>`.
+
+3. To enable the Ballerina application to publish observability statistics, save a file named `Ballerina.toml` with the following content in the `<BALLERINA_PROJECT>`.
+
+    ```ballerina
+    [build-options]
+    observabilityIncluded = true
+    ```
+   
+4. For the Ballerina application to import the `ballerinax/choreo` module, open the `.bal` file (in this example, `hello_service.bal`) and at the following line at the top.
```

Comment:
Should be "add the following line. . . ." right?

Reply:
Corrected